src/similarity/similarity.py

```diff
@@ -0,0 +1,229 @@
+import librosa
+import fileinput
+from scipy.spatial import distance
+import numpy as np
+import os
+import falconn
+from multiprocessing import Pool, Process, Queue, cpu_count, Manager
+from database.song_segment import SongSegment
+
+MATCHES = 10
+BUCKET_SIZE = 60 * 100
+
+
+def flatten(l): return [item for sublist in l for item in sublist]
+
+
+def process_segment(segment, sr):
+    harmonic, percussive = librosa.effects.hpss(segment)
+
+    tempogram = librosa.feature.tempogram(
+        percussive, hop_length=1024, win_length=16)
+
+    mfcc = librosa.feature.mfcc(segment, hop_length=1024)
+
+    chromagram = librosa.feature.chroma_cqt(y=harmonic, sr=sr, hop_length=1024)
+
+    return mfcc, chromagram, tempogram
+
+
+def load_songs(songs):
+    seg_db = SongSegment()
+    segments = []
+    for song in songs:
+        song_id, filename = song
+        segments.append(_load_song(song_id, filename, seg_db, False))
+    seg_db.close()
+    return flatten(segments)
+
+
+def _load_song(song_id, filename, segments, force):
```

Comment:
Doesn't seem like force is being set to anything other than False. Couldn't it be removed?